sentry_sdk/integrations/fastapi.py

```diff
@@ -62,6 +65,22 @@ def patch_get_request_handler():
 
     def _sentry_get_request_handler(*args, **kwargs):
         # type: (*Any, **Any) -> Any
+        dependant = kwargs.get("dependant")
+        if dependant and not asyncio.iscoroutinefunction(dependant.call):
+            old_call = dependant.call
```

Comment:
It seems that `dependant.call` could be `None`: https://github.com/tiangolo/fastapi/blob/master/fastapi/dependencies/models.py#L27

Do we have to check for that here to prevent calling None as a function dow at line 80?

Reply:
Good catch! Updated!